package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-react",
-  "version": "7.34.0",
+  "version": "7.34.1",
```

Comment:
PRs must never bump versions, and this would be a minor anyways, not a patch

Reply:
Yea, but I don't know why version on `master` is 7.34.0, whilst the last released version is 7.34.1. Is there a different branch which is more up to date than the `master` branch?

Reply:
Should be fixed on a rebase.